edx_notifications/digest_timers.py

```diff
@@ -0,0 +1,47 @@
+"""
+Create and register a new NotificationCallbackTimerHandler
+"""
+import datetime
+from django.dispatch import receiver
+import pytz
+from edx_notifications.data import NotificationCallbackTimer
+from edx_notifications.signals import perform_timer_registrations
+from edx_notifications.stores.store import notification_store
+from edx_notifications.exceptions import ItemNotFoundError
+
+DAILY_DIGEST_TIMER_NAME = 'daily-digest-timer'
+WEEKLY_DIGEST_TIMER_NAME = 'weekly-digest-timer'
+
+
+@receiver(perform_timer_registrations)
+def register_digest_timers(sender, **kwargs):  # pylint: disable=unused-argument
+    """
+    Register NotificationCallbackTimerHandler.
+    This will be called automatically on the Notification subsystem startup (because we are
+    receiving the 'perform_timer_registrations' signal)
+    """
+    store = notification_store()
+
+    try:
+        store.get_notification_timer(DAILY_DIGEST_TIMER_NAME)
+    except ItemNotFoundError:
+        daily_digest_timer = NotificationCallbackTimer(
+            name=DAILY_DIGEST_TIMER_NAME,
+            callback_at=datetime.datetime.now(pytz.UTC),
+            class_name='edx_notifications.callbacks.NotificationDigestMessageCallback',
+            is_active=True,
+            periodicity_min=24 * 60
```

Comment:
Maybe for better readability 24 * 60 should be a global const like DAILY_IN_MINS